samples/development-frameworks/SqlServerOnDocker/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       SA_PASSWORD: "Alaska2017"
       ACCEPT_EULA: "Y"
     healthcheck:
-      test: sqlcmd -S db1.internal.prod.example.com -U SA -P 'Alaska2017' -Q 'select 1'
+      test: sqlcmd -S db1.internal.prod.example.com -U SA -P 'Alaska2017' -Q 'select 1 from AdventureWorksDW2017'
```

Comment:
Seems incorrect. Is there a table called AdventureWorksDW2017?

Reply:
@uc-msft  any feedback ?

Reply:
conflicts resolved as well